CHANGELOG.md

```diff
@@ -5,14 +5,21 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 1.0.0
+
+## 1.1.0
```

Comment:
Technisch gesproken hoort er ook een datum bij de release als we de Keep a Changelog format volgen